pom.xml

```diff
@@ -179,6 +179,32 @@
             </properties>
         </profile>
 
+        <!-- 7.3 -->
+        <profile>
+            <id>community-73-tests</id>
+            <properties>
+                <alfresco.bomDependency.artifactId>acs-community-packaging</alfresco.bomDependency.artifactId>
+                <alfresco.platform.version>7.3.0-A20</alfresco.platform.version>
+                <alfresco.share.docker.version>7.3.0-A20</alfresco.share.docker.version>
+                <alfresco.share.version>17.96</alfresco.share.version>
```

Comment:
same as above, maybe we should use 17.94 instead

Reply:
For SDK 4.5 we will use the final version of 7.3.0 Share, so that will be changed later.